SparkPost/2web2ui
# FE-1348 - Resolve high severity Dependabot alerts

[FE-1348](https://sparkpost.atlassian.net/browse/FE-1348)

### What Changed

- Uninstalls unused and unmaintained `fork-ts-checker-webpack-plugin-alt`
- Updates `terser-webpack-plugin` to use version with updated dependency
- Uninstalls `mdx-loader` which depended on a package with a security vulnerability
- Upgrades `axios` to version 0.21.1
- Removes Storybook from the repo - fixing a vulnerability would have required a major overhaul - and we aren't really using it! Good to remove dependencies we aren't using
- Manually installs `raw-loader` - this must have been a transitive dependency of one of the above previously

### How To Test

- Ensure the app continues to build
- Ensure tests in CI continue to pass
- Manually test core flows and verify they still work as expected

### To Do
